demisto_sdk/commands/content_graph/objects/repository.py

```diff
@@ -23,12 +23,15 @@ class ContentDTO(BaseModel):
     def dump(
         self, dir: DirectoryPath, marketplace: MarketplaceVersions, zip: bool = True
     ):
+        dir.mkdir(parents=True, exist_ok=True)
+        Pack.__fields__.pop("relationships_data")
```

Comment:
Please add a comment on why this is popped.

Reply:
Not needed, outdated